go/vt/vtctld/api.go

```diff
@@ -287,9 +287,17 @@ func initAPI(ctx context.Context, ts *topo.Server, actions *ActionRepository, re
 
 		if cell == "local" {
 			if *localCell == "" {
-				return nil, fmt.Errorf("local cell requested, but not specified. Please set with -cell flag")
+				cells, err := ts.GetCellInfoNames(ctx)
```

Comment:
Does this mean that `local` is a reserved cell name and no one can use it for their cells? We need to document that.